lib/bootscale/file_storage.rb

```diff
@@ -13,11 +13,13 @@ def load(load_path)
 
     def dump(load_path, cache)
       path = cache_path(load_path)
-      return if File.exist?(path)
+      raise "This should not happen" if File.exist?(path)
```

Comment:
Hum, good point. What I just said is not up to date. :+1: but we'd need to assert it via another integration test.